internal/verifier/verifier.go

```diff
@@ -231,11 +186,13 @@ func (verifier *ProvenanceIRVerifier) Verify() (VerificationResult, error) {
 }
 
 // verifyBinarySHA256Digest verifies that the binary SHA256 in this provenance is contained in the given reference binary SHA256 digests (in want).
-func (got *ProvenanceIR) verifyBinarySHA256Digest(want common.ReferenceValues) (VerificationResult, error) {
+func verifyBinarySHA256Digest(want common.ReferenceValues, got *common.ProvenanceIR) (VerificationResult, error) {
 	result := NewVerificationResult()
 
-	if len(got.BinarySHA256Digests) != 1 {
-		return result, fmt.Errorf("got not exactly one actual binary SHA256 digest (%v)", got.BinarySHA256Digests)
+	gotBinarySHA256Digest, err := got.GetBinarySHA256Digest()
```

Comment:
nit: 

```suggestion
	got, err := got.GetBinarySHA256Digest()
```

Since `want` is redeclared below, I wonder if it make sense to redeclare `got` too for better readability. (and to follow Golang's guideline for shorter variable names!)

Reply:
Ah, we cant do that because we use got. to declare `gotBinarySHA256Digest`. I renamed want to wantBinarySHA256Digest to make it more uniform. But not shorter variable names, I am afraid.